pulsar/manifest.json

```
@@ -0,0 +1,35 @@
+{
+  "display_name": "pulsar",
+  "maintainer": "ming.luo@kesque.com",
+  "manifest_version": "1.0.0",
+  "name": "pulsar",
+  "metric_prefix": "kesque.pulsar.",
+  "metric_to_check": "",
```

```suggestion
  "metric_to_check": "kesque.pulsar.consumer.available_permits",
```
This will mark the pulsar integration tile as installed when detected.